Azure/connectors/azure_eventhub.py

```diff
@@ -107,25 +104,33 @@ def get_records_from_message(message: EventData) -> list[Any]:
         """
         Return the records according to the body of the message
         """
-        body = message.body_as_json()
-        if isinstance(body, list):  # handle list of events
-            return body
-        elif isinstance(body, dict) and "records" in body:  # handle wrapped events
-            return cast(list[Any], body.get("records", []))
-        elif body.get("type") == "heartbeat":  # exclude heartbeat messages
-            return []
-        else:
-            return [body]
+        try:
+            body = message.body_as_json()
+            if isinstance(body, list):  # handle list of events
+                return body, "json"
+            elif isinstance(body, dict) and "records" in body:  # handle wrapped events
+                return cast(list[Any], body.get("records", [])), "json"
+            elif body.get("type") == "heartbeat":  # exclude heartbeat messages
+                return [], "json"
+            else:
+                return [body], "json"
+        except:
+            body = message.body_as_str()
+            return [body], "str"
```

Comment:
Can you change the signature of the method:
`list[Any]` -> `tuple[list[Any], str]`

Reply:
Yes thank you